oms-all-deploy/azuredeploy.json

```diff
@@ -80,1129 +80,114 @@
       "metadata": {
         "description": "Specify the region for your Automation account"
       }
-    }
-  },
-  "variables": {
-    "psModules": {
-      "azureRmProfile": {
-        "name": "AzureRm.Profile",
-        "url": "https://devopsgallerystorage.blob.core.windows.net/packages/azurerm.profile.1.0.11.nupkg"
-      },
-      "azureRmResources": {
-        "name": "AzureRm.Resources",
-        "url": "https://devopsgallerystorage.blob.core.windows.net/packages/azurerm.resources.2.0.2.nupkg"
-      },
-      "azureStorage": {
-        "name": "Azure.Storage",
-        "url": "https://devopsgallerystorage.blob.core.windows.net/packages/azure.storage.1.1.5.nupkg"
-      },
-      "azureRmStorage": {
-        "name": "AzureRm.Storage",
-        "url": "https://devopsgallerystorage.blob.core.windows.net/packages/azurerm.storage.1.1.3.nupkg"
-      },
-      "azureRmOperationalInsights": {
-        "name": "AzureRm.OperationalInsights",
-        "url": "https://devopsgallerystorage.blob.core.windows.net/packages/azurerm.operationalinsights.1.0.9.nupkg"
-      },
-      "azureRmSiteRecovery": {
-        "name": "AzureRm.SiteRecovery",
-        "url": "https://devopsgallerystorage.blob.core.windows.net/packages/azurerm.siterecovery.1.1.10.nupkg"
-      },
-      "azureRmRecoveryServices": {
-        "name": "AzureRm.RecoveryServices",
-        "url": "https://devopsgallerystorage.blob.core.windows.net/packages/azurerm.recoveryservices.1.1.3.nupkg"
-      },
-      "azureRmBackup": {
-        "name": "AzureRm.Backup",
-        "url": "https://devopsgallerystorage.blob.core.windows.net/packages/azurerm.backup.1.0.9.nupkg"
-      },
-      "azureRmCompute": {
-        "name": "AzureRm.Compute",
-        "url": "https://devopsgallerystorage.blob.core.windows.net/packages/azurerm.compute.1.3.3.nupkg"
-      },
-      "azureRmAutomation": {
-        "name": "AzureRm.Automation",
-        "url": "https://devopsgallerystorage.blob.core.windows.net/packages/azurerm.automation.1.0.11.nupkg"
+    },
+    "_assetLocation": {
```

Comment:
That I get. In this case we were aiming to expose as few paramaters as possible to the user to simplify the onboarding process.